addons/website/static/src/scss/website.scss

```diff
@@ -845,7 +845,7 @@ $-transition-duration: 200ms;
         > ul, > .o_mega_menu { // remove dropdown-menu default style as it is nested in another one
             position: static;
             float: none;
-            display: block;
+            display: inline;
```

Comment:
You should rather unset the transform on `.o_mega_menu.o_mega_menu_container_size` when it's within a `.o_extra_menu_items` element. It's not obvious that `display: inline` disables the transform and it could break custom or other CSS code based on the display property of the mega menu.